protocol/query/method/message/data/dataCloseRollCall.json

```diff
@@ -29,12 +29,11 @@
 	"required": ["object", "action", "update_id", "closes", "end", "attendees"],
```

Comment:
I think "end" should be updated to "closed_at" here. Same thing for the updated fields in dataCreateRollCall.json and dataOpenRollCall.json.